apis/supervisor/idp/v1alpha1/types_githubidentityprovider.go.tmpl

```diff
@@ -52,8 +52,12 @@ type GitHubIdentityProviderStatus struct {
 // GitHubAPIConfig allows configuration for GitHub Enterprise Server
 type GitHubAPIConfig struct {
 	// Host is required only for GitHub Enterprise Server.
-	// Defaults to using GitHub's public API (github.com).
-	// Do not specify a protocol or scheme since 'https://' will always be used.
+	// Defaults to using GitHub's public API ("github.com").
+	// Do not specify a protocol or scheme since "https://" will always be used.
+	// Port is optional. Do not specify a path, query, fragment, or userinfo.
+	// Only domain name or IP address, subdomains (optional), and port (optional).
+	// IPv4 and IPv6 are supported. If using an IPv6 address with a port, you must enclose the IPv6 address
+	// in square brackets. Example: "[::1]:443".
```

Comment:
Do we have a test or two with IPv6 with and without the brackets?

Reply:
not yet

Reply:
We now have an IPv6-with-brackets test. Without brackets is up to `endpointaddr` to check, so I didn't feel it was necessary at the controller level.